pontoon/base/templates/locale_selector.html

```diff
@@ -27,7 +27,7 @@
       {% for l in locales %}
       <li class="clearfix{% if locale and locale == l %} current{% endif %}{% if l and l.chart %} filter{% endif %}">
         {% if l and l.chart %}
-        <a href="/{{ l.code|lower }}/{{ project.slug }}/" class="clearfix">
```

Comment:
You should use the `url` helper (from `pontoon.base.templatetags.helpers`) to generate these URLs by their name instead of building them manually. It helps protect us in case we want or need to change up the URL structure.